packages/jest-cli/src/reporters/DefaultTestReporter.js

```diff
@@ -139,6 +140,37 @@ class DefaultTestReporter {
         `${PENDING_COLOR(`${print('test', pendingTests)} skipped`)}, `;
     }
 
+    let snapshotsAdded = 0;
+    let snapshotsFiles = 0;
+    let snapshotsMatched = 0;
+    let snapshotsRemoved = 0;
+    aggregatedResults.testResults.forEach(result => {
+      if (result.snapshotsAdded) {
+        snapshotsFiles++;
+      }
+      snapshotsAdded += result.snapshotsAdded;
+      snapshotsMatched += result.snapshotsMatched;
+      snapshotsRemoved += result.snapshotsRemoved;
+    });
+    if (snapshotsAdded || snapshotsMatched) {
```

Comment:
shouldn't this also be printed when snapshots were removed? I know we don't have this capability yet, but it still seems like a bug.